MoesifNetCore3Example/Controllers/StudentController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
nitpick: can we name the file something like AsyncStudentController

Reply:
Refactored the file name.